examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -11,6 +11,16 @@ proxyAddr: ""
 # Comma-separated list of roles to enable (any of: kube,db,app)
 roles: "kube"
 
+################################################################
```

Comment:
Shouldn't it be documented at https://raw.githubusercontent.com/gravitational/teleport/master/docs/pages/reference/helm-reference/teleport-kube-agent.mdx ? Since this PR is related with `teleport-kube-agent`.

Reply:
Yup, my bad I pointed to the wrong chart